torch/nn/modules/module.py

```diff
@@ -461,3 +461,10 @@ def __repr__(self):
             tmpstr = tmpstr + '  (' + key + '): ' + modstr + '\n'
         tmpstr = tmpstr + ')'
         return tmpstr
+
+    def __dir__(self):
+        _moduel_attrs = dir(self.__class__)
+        _parameters = self._parameters.keys()
+        _modules = self._modules.keys()
+        _attrs = self.__dict__.keys()
+        return _moduel_attrs + _parameters + _modules + _attrs
```

Comment:
1. you have a typo in module
2. it should be `super(Module, self).__dir__() + _parameters + _modules + _buffers`